docs/modules/ROOT/pages/index.adoc

```diff
@@ -85,11 +208,11 @@ Endorsements are COSE objects (see link:++https://c2pa.org/specifications/specif
 
 |X.509 SubjectPublicKeyInfo
 |1
-|A https://datatracker.ietf.org/doc/html/rfc5280#section-4.1[DER-encoded ASN.1 SubjectPublicKeyInfo] structure (as a CBOR `bstr`), from the _endorsee’s_ X.509 certificate that will be used for signing the claims of assets that this endorsement is expected to be used with.
-|The value of the credential shall be compared (bitwise identical) to the SubjectPublicKeyInfo structure of the certificate located in the `x5chain` or `33` (integer) COSE header of the active manifest Claim Signature. (See link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#x509_certificates++[Section 14.6, “X.509 Certificates,” of the C2PA 2.0 Technical Specification].)
+|A https://datatracker.ietf.org/doc/html/rfc5280#section-4.1[DER-encoded ASN.1 SubjectPublicKeyInfo] structure (as a CBOR `bstr`), from the _endorsee’s_ X.509 certificate that will be used for signing the claims of _<<_c2pa_asset,C2PA assets>>_ that this _<<_endorsement,endorsement>>_ is expected to be used with.
```

Comment:
To bring this in line with @paulengland's suggested changes to the identity assertion, I think this should be a hash of the endorsee's certificate. This would also prevent a situation where an endorsee re-uses a public/private key pair to get a new certificate issued.

Reply:
@scouten-adobe be sure to update validation to match this change.